src/server.h

```diff
@@ -244,8 +244,13 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 			{ return m_shutdown_requested; }
 
 	// request server to shutdown
-	inline void requestShutdown(void)
-			{ m_shutdown_requested = true; }
+	inline void requestShutdown() { m_shutdown_requested = true; }
```

Comment:
m_shutdown_ask_reconnect defaults to false, and m_shutdown_msg to "", and if m_shutdown_msg is "", we read the setting kick_msg_shutdown.